### PR TITLE
6.1.1: Update sensuctl dump and prune hook FQN and type

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Updates hook fully qualified name and type in supported resource types tables in https://docs.sensu.io/sensu-go/latest/sensuctl/back-up-recover/#supported-resource-types and https://docs.sensu.io/sensu-go/latest/sensuctl/create-manage-resources/#supported-resource-types
- Adds notes after the supported resource types tables to identify the different fully qualified name and type for hooks for 6.1.0.

## Motivation and Context
Support 6.1.1 release based on changes in https://github.com/sensu/sensu-go/pull/4057

### CODE REVIEW

File: content/sensu-go/6.1/sensuctl/back-up-recover.md [hunk 1]
```diff
@@ -174,13 +174,17 @@ sensuctl describe-type all
...
+  core/v2.HookConfig             hooks                 core/v2             HookConfig           true
   core/v2.Mutator                mutators              core/v2             Mutator              true
   core/v2.Role                   roles                 core/v2             Role                 true
   core/v2.RoleBinding            rolebindings          core/v2             RoleBinding          true
   core/v2.Silenced               silenced              core/v2             Silenced             true  
 {{< /code >}}
 
+{{% notice note %}}
+**NOTE**: In Sensu 6.1.0, for hooks, the fully qualified name is `core/v2.Hook` and the type is `Hook`.
+{{% /notice %}}
```
Comment (palourde) [inline content/sensu-go/6.1/sensuctl/back-up-recover.md:186]: So in 6.1.0 and previous versions, it wouldn't even work at all with `core/v2.Hook`, so I'm wondering if we should modify this note somehow, so users don't expect it to work?

File: content/sensu-go/6.1/sensuctl/back-up-recover.md [hunk 2]
```diff
@@ -174,13 +174,17 @@ sensuctl describe-type all
...
-  core/v2.Hook                   hooks                 core/v2             Hook                 true
+  core/v2.HookConfig             hooks                 core/v2             HookConfig           true
   core/v2.Mutator                mutators              core/v2             Mutator              true
   core/v2.Role                   roles                 core/v2             Role                 true
   core/v2.RoleBinding            rolebindings          core/v2             RoleBinding          true
   core/v2.Silenced               silenced              core/v2             Silenced             true  
 {{< /code >}}
 
+{{% notice note %}}
+**NOTE**: In Sensu 6.1.0, for hooks, the fully qualified name is `core/v2.Hook` and the type is `Hook`.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/sensuctl/back-up-recover.md]:
```suggestion
**NOTE**: In Sensu 6.1.0, `sensuctl dump` does not work with hooks.
```

File: content/sensu-go/6.1/sensuctl/create-manage-resources.md [hunk 3]
```diff
@@ -475,13 +475,17 @@ sensuctl describe-type all
...
-  core/v2.Hook                   hooks                 core/v2             Hook                 true
+  core/v2.HookConfig             hooks                 core/v2             HookConfig           true
   core/v2.Mutator                mutators              core/v2             Mutator              true
   core/v2.Role                   roles                 core/v2             Role                 true
   core/v2.RoleBinding            rolebindings          core/v2             RoleBinding          true
   core/v2.Silenced               silenced              core/v2             Silenced             true  
 {{< /code >}}
 
+{{% notice note %}}
+**NOTE**: In Sensu 6.1.0, for hooks, the fully qualified name is `core/v2.Hook` and the type is `Hook`.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/sensuctl/create-manage-resources.md]:
```suggestion
**NOTE**: In Sensu 6.1.0, `sensuctl prune` does not work with hooks.
```

**Review by palourde — APPROVED** [review]
👌

**Pull request merged**